steamtinkerlaunch

```diff
@@ -5709,6 +5713,7 @@ function AllSettingsEntriesDummyFunction {
 --field="     $GUI_RUNSBS!$DESC_RUNSBS ('RUNSBS')":CHK "${RUNSBS/#-/ -}" `#CAT_Misc` `#SUB_Checkbox` `#MENU_GAME` \
 --field="     $GUI_SDLUSEWAYLAND!$DESC_SDLUSEWAYLAND ('SDLUSEWAYLAND')":CHK "${SDLUSEWAYLAND/#-/ -}" `#CAT_Misc` `#SUB_Checkbox` `#MENU_GAME` \
 --field="     $GUI_STLRAD_PFTST!$DESC_STLRAD_PFTST ('STLRAD_PFTST')":CBE "$(cleanDropDown "${STLRAD_PFTST/#-/ -}" "none!gpl!sam!rt!emulate_rt!rtwave64!video_decode")"  `#CAT_Misc` `#MENU_GAME` \
+--field="     $GUI_STEAMDECK_AUTOUP!$DESC_STEAMDECK_AUTOUP ('STEAMDECK_AUTOUP')":CHK "${STEAMDECK_AUTOUP/#-/ -}" `#CAT_Misc` `#SUB_Checkbox` `#MENU_GLOBAL` \
```

Comment:
The actual code here is absolutely fine, it's just the `GUI_STEAMDECK_AUTOUP` and `DESC_STEAMDECK_AUTOUP` need to be added to the langfiles. For initial testing you can just put it in `lang/english.txt`, until the wording is finalised, and then copypaste it into the other files afterwards (I want to make a script for it at some point, to sync langfiles, I've just been lazy.)

Don't see any issues apart from that though. It won't crash either, but the checkboxes would show up empty.

Reply:
You're right, thanks for pointing it out ! I was looking at the code and didn't know where all those variables were actually being set !